### ISSUE TITLE
Secure hosting for Sefaria_linker.js

### ISSUE THREAD

**Issue by hbrandes** [Sefaria/Sefaria-Project#275]
We are trying to incorporate Sefaria links into our courses on canvas LMS.  This pages are hosted securely and served over https.  Unfortunately because  Sefaria's linker.js is only hosted on http://sefaria.org/ we are brewers are blocking the script as insecure with messages such as "Mixed Content: The page was loaded over HTTPS, but requested an insecure script 'http://sefaria.org/linker.js'. This request has been blocked; the content must be served over HTTPS."  To avoid this we are temporarily hosting a copy on our https server, but it this won't update as changes are made to the script over time, so we would rather get them directly from the source.

Sefaria already has a security certificate and hosts a secure version of the site, so I am hoping it would be trivial to have https://sefaria.org/linker.js added as an option as well.

Thanks so much,

Hyim

**Comment by EliezerIsrael** [comment]
Hyim - 

Great to hear that you're working to integrate the linker into your LMS.  
Have you tried changing the protocol in the initial load, substituting `https` for `http`?

i.e:
```
    <script type="text/javascript" charset="utf-8" src="https://sefaria.org/linker.js"></script>
    <script>
        sefaria.link();
    </script>
```

If that doesn't work, please let me know how it fails.

**Comment by hbrandes** [comment]
Thanks for the quick response.  When I tried it initially it failed with a 404 error (I then tried loading it manually and got the same 404 error), but when I tried it again today, after your response it worked, I am not sure what I did wrong the first time but thanks.  

Since it does in fact work over https, can I suggest that the suggested embed code on https://github.com/Sefaria/Sefaria-Project/wiki/Sefaria-Auto-Linker be amended to the secure version (or at least that it be made clear that it is an option).  

Thanks again
